src/mixins/components/input-styles.scss

```diff
@@ -0,0 +1,53 @@
+@mixin input-styles($element-type) {
```

Comment:
Is fine but I'd just name this `input` or something. It's a given that a mixin mixes in styles.